diagnostic.md

```diff
@@ -49,7 +53,9 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+@function tint($color, percentage) {
+    @return mix($pure-white, $cornflower, $20%);
+}
```

Comment:
The syntax for this function is incorrect; however, since you didn't need to write the function for this question, I won't take points off.